pymongo/topology.py

```diff
@@ -213,7 +213,7 @@ def _select_servers_loop(self, selector, timeout, address):
             # No suitable servers.
             if timeout == 0 or now > end_time:
                 raise ServerSelectionTimeoutError(
-                    self._error_message(selector))
+                    self._error_message(selector)+", Timeout: "+str(timeout)+"s, "+repr(self.description))
```

Comment:
I think we should prefix the `self.description` repr with something like `Topology Description: ` so the overall error message has a structure like this: `ServerSelectionTimeoutError: <reason>, Timeout: <duration>s, Topology Description: <topology repr>`.